pages/spicedb/modeling/attributes.mdx

```diff
@@ -22,10 +22,9 @@ Feature flag authorization can be enabled with boolean attributes.
 [Wildcards](/spicedb/concepts/schema#wildcards) are a way to implement boolean attributes.
 Wildcards modify a type so that a relationship can be written to all objects of a resource type but not individual objects.
 
-In the example below, the schema enforces the following authorization logic: a user can only view a document if the user is related to the document as viewer and editing is enabled for the document.
-To enable document editing in the example below, you need to establish a wildcard relationship that connects all users to the document using the ```edit_enabled``` wildcard relation.
-A user will have ```edit``` permission on the document if they are related to the document as an ```editor``` and they relate to the document through ```edit_enabled```.
+In the example below, the schema enforces the following authorization logic: a user will have ```edit``` permission on the document if they are related to the document as an ```editor``` and they relate to the document through ```edit_enabled```.
```

Comment:
I've never seen inlined-triple-backtick. does it differ from single backtick?

I guess you're cribbing from what's around it :thinking: